website/api/blockchain-integration/contracts/verifier.md

```diff
@@ -4,6 +4,14 @@ To utilize the results generated by a [zkVM][term-zkvm] program in your on-chain
 
 You can verify a RISC Zero [Groth16Receipt] using a `verify()` call to an [IRiscZeroVerifier][IRiscZeroVerifier] contract.
 
+## Verification Cost
+
+SNARK verification via the `verifyProof` function on the [RISCZeroGroth16Verifier][RISCZeroGroth16Verifier] contract costs around 215k gas. However, you'll notice that a transaction calling `verify()` via the [RiscZeroVerifierRouter](#verifer-router) costs [~316k gas][verify-sepolia-etherscan]. 
```

Comment:
hmm... i think tehse numbers are ethereum-specific, while the rest of the page is chain-agnostic